docs/source/delta-sharing.md

```diff
@@ -0,0 +1,239 @@
+
+  // timestamps as formatted timestamp
+  spark.read.format("deltaSharing")
+    .option("readChangeFeed", "true")
+    .option("startingTimestamp", "2024-01-18 05:45:46")
+    .option("endingTimestamp", "2024-01-18 12:00:00")
+    .load(tablePath)
+  
+  // providing only the startingVersion/timestamp
+  spark.read.format("deltaSharing")
+    .option("readChangeFeed", "true")
+    .option("startingVersion", 0)
+    .load(tablePath)
+  ```
+
+## Streaming
+
+Delta Sharing Streaming is deeply integrated with [Spark Structured Streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html) through `readStream`, 
+and able to connect with any sink that is able to perform `writeStream`.
+
+Once the data provider shares a table with history, the recipient can perform a streaming query on the table.
+When you load a Delta Sharing table as a stream source and use it in a streaming query, the query processes all of the data present in the shared table as well as any new data that arrives after the stream is started.
+
+```scala
+val tablePath = "<profile-file-path>#<share-name>.<schema-name>.<table-name>"
+
+spark.readStream.format("deltaSharing").load(tablePath)
+```
+
+Delta Sharing Streaming supports the following functionalities in the same way as Delta Streaming: [Limit input rate](delta-streaming.md#limit-input-rate), 
+[Ignore updates and deletes](delta-streaming.md#ignore-updates-and-deletes), [Specify initial position](delta-streaming.md#specify-initial-position)
+
+In addition, `maxVersionsPerRpc` is provided to decide how many versions of files are requested from the server in every Delta Sharing rpc. This is to help
+reduce the per rpc workload and make the Delta Sharing streaming job more stable. 
+Especially when the streaming resumes from a checkpoint and a lots of new versions are accumulated on the shared table on the server. The default is 100.
+
+.. note:: Trigger.AvailableNow is not supported in Delta Sharing Streaming.
+
+## Read Advanced <Delta> Features in Delta Sharing
+In order to support advanced <Delta> features in Delta Sharing, "Delta Format Sharing" is introduced since delta-sharing-client 1.0 and delta-sharing-spark 3.1, 
+in which the actions of a shared table is returned in <Delta> format, then a <Delta> library is leveraged to read data.
+
+Please do remember to set the spark configurations mentioned in [_](delta-batch.md#sql-support) in order to read shared tables with DeletionVectors and ColumnMapping. 
+
+| Read Table Feature | Available since version |
+| - | - |
+| Deletion Vectors(#delta-deletion-vectors.md) | 3.1.0 | 
+| ColumnMapping(#delta-column-mapping.md) | 3.1.0 |
+
+Batch queries can be performed as is, because it can automatically resolve the responseFormat based on the table features of the shared table.
+An additional option `responseFormat=delta` needs to be set for cdf and streaming queries when reading shared tables with DeletionVectors or ColumnMapping enabled.
```

Comment:
DV, CM

Reply:
updated to Deletion Vectors and Column Mapping, it doesn't seem like delta is using any abbreviation.